University of Notre Dame/NWICG_NDCMS/NWICG_Eddie.yaml

```
@@ -0,0 +1,28 @@
+Disable: false
+GroupDescription: NWICG_Eddie
+GroupID: 
+Production: true
+Resources:
+  NWICG_Eddie:
+    Active: true
+    ContactLists:
+      Administrative Contact:
+        Primary:
+          ID: 1a6defbcb27777eae626bbb0ae9f4d0a281dd641
+          Name: Irena Johnson
+      Security Contact:
+        Primary:
+          ID: 1a6defbcb27777eae626bbb0ae9f4d0a281dd641
+          Name: Irena Johnson
+    Description: The FQDN is "eddie.crc.nd.edu"
+    Disable: false
+    FQDN: eddie.crc.nd.edu
+    ID: 
+    Services:
+      SQUID:
+        Description: Squid Server
+        Details:
+          hidden: false
+    VOOwnership:
+      CMS: 
```

Irena, even if owned by a single VO, we need the percentage to add up to 100. [See these lines](https://github.com/opensciencegrid/topology/blob/master/template-resourcegroup.yaml#L85-L90).

```suggestion
      CMS: 100
```